### PR TITLE
Fix icons in dashboard activity

### PR DESCRIPTION
**Pull request by ca3tiel**
Add New Icons to Dashboard Activity
-Remove MCI and MTN Icon
-Add Premium Servers New Icon
-Add 10Gbps Icon
-Add Gray Lock and Green Unlock Icons
-Some Minor Changes

### CODE REVIEW

File: app/src/main/java/io/nekohasekai/sagernet/vpn/DashboardActivity.kt [hunk 1]
```diff
@@ -182,8 +180,7 @@ class DashboardActivity : BaseThemeActivity(),
 
         powerIcon = findViewById(R.id.laPulseButton)
         ivAll = findViewById(R.id.ivAll)
-        ivMtn = findViewById(R.id.ivMtn)
-        ivMci = findViewById(R.id.ivMci)
+        ivPremiumServers = findViewById(R.id.ivPremiumServers)
```
Comment (kamrava) [inline app/src/main/java/io/nekohasekai/sagernet/vpn/DashboardActivity.kt]: Use binding

File: app/src/main/java/io/nekohasekai/sagernet/vpn/DashboardActivity.kt [hunk 2]
```diff
@@ -248,7 +243,7 @@ class DashboardActivity : BaseThemeActivity(),
         // Set an OnClickListener for IVall
         ivAll.setOnClickListener {
             ivAllClicked = !ivAllClicked // Toggle the IVall click state
-            updateIVAllIcon() // Update the IVall icon
+            updateivAllIcon() // Update the IVall icon
```
Comment (kamrava) [inline app/src/main/java/io/nekohasekai/sagernet/vpn/DashboardActivity.kt]: Change it to `updateIvAllIcon()`

File: app/src/main/java/io/nekohasekai/sagernet/vpn/DashboardActivity.kt [hunk 3]
```diff
@@ -262,59 +257,31 @@ class DashboardActivity : BaseThemeActivity(),
...
                 transaction.commit()
             }
-            // Reset IVMTN and IVMCI click states
-            ivMtnClicked = false
-            ivMciClicked = false
-            updateIVMtnIcon() // Update the IVMTN icon
-            updateIVMciIcon() // Update the IVMCI icon
+            // Reset PremiumServers click states
+            ivPremiumServersClicked = false
+            updateivPremiumServersIcon() // Update the PremiumServers icon
```
Comment (kamrava) [inline app/src/main/java/io/nekohasekai/sagernet/vpn/DashboardActivity.kt]: Change it to `updateIvPremiumServersIcon()`

File: app/src/main/java/io/nekohasekai/sagernet/vpn/DashboardActivity.kt [hunk 4]
```diff
@@ -262,59 +257,31 @@ class DashboardActivity : BaseThemeActivity(),
...
+        ivPremiumServers.setOnClickListener {
+            ivPremiumServersClicked = !ivPremiumServersClicked // Toggle the ivPremiumServers click state
+            updateivPremiumServersIcon() // Update the ivPremiumServers icon
             // Show/hide the MyFragment based on the click state
-            fragmentContainer.visibility = if (ivMtnClicked) View.VISIBLE else View.INVISIBLE
-            if (ivMtnClicked) {
-                AppRepository.filterServersByTag("mtn")
+            fragmentContainer.visibility = if (ivPremiumServersClicked) View.VISIBLE else View.INVISIBLE
+            if (ivPremiumServersClicked) {
+                AppRepository.filterServersByTag("premiumservers")
```
Comment (kamrava) [inline app/src/main/java/io/nekohasekai/sagernet/vpn/DashboardActivity.kt]: `premiumservers` => `premium` is enough

Comment (kamrava) [reply app/src/main/java/io/nekohasekai/sagernet/vpn/DashboardActivity.kt]: Better to write a new method called `filterByPremiumServers()` and just filter all servers that has `is_free=false`

Comment (ca3tiel) [reply app/src/main/java/io/nekohasekai/sagernet/vpn/DashboardActivity.kt]: do this part together

**Pull request merged**